flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/job/JobDetailsInfo.java

```diff
@@ -165,6 +165,54 @@ public int hashCode() {
 		return Objects.hash(jobId, name, isStoppable, jobStatus, startTime, endTime, duration, now, timestamps, jobVertexInfos, jobVerticesPerState, jsonPlan);
 	}
 
+	public JobID getJobId() {
+		return jobId;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public boolean isStoppable() {
```

Comment:
can you try annotating this one with `@JsonIgnore` and seeing what is sent?

Jackson analyzes getters and this may mess with the resulting JSON.

Reply:
see `ResponseBody` javadocs:
```
When adding methods that are prefixed with {@code get} make sure to annotate them with {@code @JsonIgnore}.
```

Reply:
nice! I fixed it